setup.py

```diff
@@ -1,3 +1,12 @@
+##############################################################################
+# MSIBI: A package for optimizing coarse-grained force fields using multistate
+# iterative Boltzmann inversion.
+#
+# Authors: Christoph Klein
+# Contributors: Tim Moore, Davy Yue
+#
+##############################################################################
+
```

Comment:
For the module level doc strings, I was suggesting mimicking what packages like MDTraj do where every file has a header with the project description, copyright, license and a list of people who wrote THIS particular file and contributed to it - not the package as a whole. So for example, remove all of your changes from setup.py and add this header to all the .py files within msibi and you can add yourself to the ones that you contributed relevant code too. __However, please do this in a separate pull request as this will become quite nasty to review.__